### PR TITLE
Auto merge PRs to branch-24.06 from branch-24.04 [skip ci]

### PR DESCRIPTION
**Pull request by NvTimLiu**
As the title

### CODE REVIEW

File: examples/ML+DL-Examples/Spark-cuML/pca/pom.xml [hunk 1]
```diff
@@ -21,7 +21,7 @@
     <groupId>com.nvidia</groupId>
     <artifactId>PCAExample</artifactId>
     <packaging>jar</packaging>
-    <version>24.04.0-SNAPSHOT</version>
+    <version>24.06.0-SNAPSHOT</version>
```
Comment (jlowe) [inline examples/ML+DL-Examples/Spark-cuML/pca/pom.xml:24]: Changing to 24.06 on the 24.04 branch?

File: examples/UDF-Examples/RAPIDS-accelerated-UDFs/src/main/cpp/CMakeLists.txt [hunk 2]
```diff
@@ -16,7 +16,7 @@
 
 cmake_minimum_required(VERSION 3.23.1 FATAL_ERROR)
 
-file(DOWNLOAD https://raw.githubusercontent.com/rapidsai/rapids-cmake/branch-24.04/RAPIDS.cmake
+file(DOWNLOAD https://raw.githubusercontent.com/rapidsai/rapids-cmake/branch-24.06/RAPIDS.cmake
```
Comment (jlowe) [inline examples/UDF-Examples/RAPIDS-accelerated-UDFs/src/main/cpp/CMakeLists.txt:19]: Downloading 24.06 on 24.04 branch?

**Review by jlowe — COMMENTED** [review]
This PR is marked skip ci but has a bunch of changes we would want to run through CI.  I suspect there's much more that was pushed to this PR than what was intended.

**Comment by NvTimLiu** [comment]
Sorry I pushed the wrong branch, close this PR

**Pull request closed without merging**